frontend/src/high-dimensional/ui/Chart.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+    <v-card hover class="visual-dl-page-charts">
+        <div ref="chartBox" class="visual-dl-chart-box" :style="computedStyle">
+        </div>
+    </v-card>
+</template>
+
+<script>
+import echarts from 'echarts';
+
+export default {
+    props: ['config', 'displayWordLabel', 'searchText', 'embedding_data'],
+    data() {
+        return {
+            width: 900,
+            height: 600,
+        };
+    },
+    computed: {
+        computedStyle() {
+            return 'height:' + this.height + 'px;'
+                + 'width:' + this.width + 'px;';
+        }
+    },
+    created() {
+
+    },
+    mounted() {
+        this.createChart();
+        this.setChartsOptions();
+        this.setDisplayWordLabel();
+    },
+    watch: {
+        embedding_data: function(val) {
+            this.myChart.setOption({
+                series: [{
+                    // Grab the 'matched' series data
+                    name: 'all',
+                    data: val
+                }]
+            });
+        },
+        displayWordLabel: function(val) {
+            this.setDisplayWordLabel()
+        },
+        searchText: function(val) {
+            var matched_words = []
+            if (val != '') {
+                val = val.toLowerCase()
+
+                function hasPrefix(value) {
+                    var word = value[2]
```

Comment:
Is this because '2' is the label of the data, like 'yellow' in [10.0, 8.04, "yellow"] ?

Reply:
That's is correct. This way the EChart can plot the graph. EChart demo code uses this pattern.